### DISCUSSION TITLE
Generate preview images with standard prompts on model upload

### DISCUSSION THREAD

**Discussion by JustMaier** [civitai/civitai#15] [Ideas]
This idea comes from Reddit User Terrariyum:

> It would be really cool if you also included images for each model that all use a similar prompt, e.g. Keanu Reeves, Emma Watson, the Sidney opera house, and a Delorean car. For most models, at least one of those 4 examples will showcase its abilities well (with exceptions like deep space diffusion).
> 
> That would make it easy to compare the models. I think it's fine, even helpful, if those images are cherry-picked so that every model is presented in the best light. It's not necessary for every sample image to use the same seed, CFG, etc. For example, it's helpful to see what the "Jinx from Arcane" model does with "Emma Watson" and not just "Jinx".

What I believe this would involve:
- On upload to Civitai main after the model file has been security scanned, Civitai main would request a default set of prompts from an Image Generation service (perhaps something like [StableHorde](https://stablehorde.net/))
- The Image Generation service would respond with the images and they'd be connected to the Civitai Model as `StandardPreview` images
- The `StandardPreview` images would then be shown on the Model page so that users could compare between the various models
- A new Model Comparison feature could be added that would allow users to select a standard prompt and see what the results are from each Model.

**Comment by Trahloc** [comment]
This would also provide a set of images which could always be visible for those who want to browse while at work since they can be trusted SFW due to being generated with heavy anti-NSFW tags.  Plus for folks like myself it'd be nice to have a standard image for use when deciding what model/lora/TI to use.

**Comment by zielperson** [comment]
This has my vote, but please let us not use celebs.

Man, woman, camera, tv.. ok, sorry.

Joking aside, here are ten:

Young man , young woman
old man, old woman
cat, dog
car, bicycle
house, castle

**Reply by WASasquatch** [reply]
> This has my vote, but please let us not use celebs.
> 
> Man, woman, camera, tv.. ok, sorry.
> 
> Joking aside, here are ten:
> 
> Young man , young woman old man, old woman cat, dog car, bicycle house, castle

I suggest using wildcards there, like my nationality list for [Noodle Soup Prompts](https://github.com/WASasquatch/noodle-soup-prompts) ([Reference](https://github.com/WASasquatch/noodle-soup-prompts/wiki/Terminology-Reference)) or converted set for [SD Dynamic Prompt](https://github.com/adieyal/sd-dynamic-prompts) wildcards. 

That way one could set up something like `A __nationality__ __gender__` and be more representative of the people of the world. Which could give you something like `A Lebanese man` or `A Nigerian woman` etc, etc.
